src/sources/file_descriptors/file_descriptor.rs

```diff
@@ -97,12 +101,12 @@ impl SourceConfig for PipeConfig {
 }
 
 pub fn pipe_source(
```

Comment:
🤔 Thinking about if this should be renamed to file_descriptor_source as well.

Reply:
🤦 Definitely